apps/pm-user/src/components/organisms/appSkeleton/appSkeleton.module.scss

```diff
@@ -11,17 +11,24 @@
   height: calc(100% - 60px);
   display: flex;
   justify-content: space-between;
+  overflow: hidden;
```

Comment:
If you put overflow:hidden... only sidebars will look good.
This approach will hide center content and will not allow users to scroll...

So, this approach does not work for us.

Reply:
ok will remove it